web/main/templates/admin/reporting/index.html

```diff
@@ -111,6 +111,8 @@ <h1>Usage dashboard</h1>
         <th>Verified professors
             <small>
                 <a href="{% url "admin:reporting_professor_changelist" %}?{{ date_query }}">View as list »</a>
+                <a href="{% url "admin:reporting_professor_changelist" %}?{{ date_query }}&amp;_csv=True">Export as csv »</a>
```

Comment:
I don't know if this matters: I'm pretty sure you can use `&` directly here, rather than `&amp;`

Reply:
Just to satisfy curiosity: why `_csv` rather than `csv`? Is it to make it especially clear this is custom logic, not part of the Django admin's own functionality?

Reply:
My editor was grumpy about `&` but I agree it's fine so I'll change it to the more readable thing!

When I was trying to figure out how to do this weird delegation thing I saw that Django internally used a pattern like this to determine whether to pop up confirmation dialogues or otherwise dispatch differently during the change list request cycle:

https://github.com/django/django/blob/main/django/contrib/admin/options.py#L1370-L1372

so I just went with that as yeah "this is a special param" rather than a query param that filters the result list.